preprocess/word2vec.py

```diff
@@ -93,9 +96,13 @@ def embed_sentence(self, sentence, max_length):
                 print(v)
                 raise ValueError("Embedding is of the wrong size!")
 
+        embedding_length = len(embedded)
+        for i in range(embedding_length, max_length):
+            embedded = np.append(embedded, self.tokens['PAD'])

```

Comment:
It is a bit strange to not give an axis number to numpy, as the default is to first flatten the arrays and then append the values....